projects/services/storage-service-typescript/src/index.ts

```diff
@@ -1,24 +1,42 @@
-import { ErrorHandler, grpc, ServiceRunner, topology, topologyPeers } from "orbs-core-library";
+import * as path from "path";
+
+import { logger, ErrorHandler, grpc, ServiceRunner, topology, topologyPeers } from "orbs-core-library";
 
 import BlockStorageService from "./block-storage-service";
 import StateStorageService from "./state-storage-service";
 
+const { NODE_NAME, NODE_ENV, LOGZIO_API_KEY, BLOCK_STORAGE_POLL_INTERVAL } = process.env;
+
 ErrorHandler.setup();
 
-const { BLOCK_STORAGE_POLL_INTERVAL } = process.env;
+logger.configure({
+  file: {
+    fileName: path.join(__dirname, "../../../../logs/storage.log")
+  },
+  logzio: {
+    apiKey: LOGZIO_API_KEY
+  },
+});
+
+if (!NODE_NAME) {
+  throw new Error("NODE_NAME can't be empty!");
+}
 
 const nodeTopology = topology();
 const peers = topologyPeers(nodeTopology.peers);
-const nodeConfig = {
-  nodeName: nodeTopology.name,
-  pollInterval: Number(BLOCK_STORAGE_POLL_INTERVAL) || 5000
-};
 
-const main = async () => {
-  await ServiceRunner.runMulti(grpc.storageServiceServer, [
-    new BlockStorageService(peers.gossip, nodeConfig),
-    new StateStorageService(peers.blockStorage, nodeConfig)
-  ], nodeTopology.endpoint);
+const blockStorageDBPath = path.resolve(path.join("../../../db", NODE_ENV || "development") + "blocks.db");
```

Comment:
Maybe we could use `BLOCK_STORAGE_DB_PATH` instead of constructing a relative path?

Reply:
Do you mean to take from `process.env` (and if it doesn't exist - to use this default value)?

Reply:
Yes.

Reply:
You got it 👍